src/providers/azure-openai/api.ts

```diff
@@ -2,14 +2,17 @@ import { ProviderAPIConfig } from '../types';
 import {
   getAccessTokenFromEntraId,
   getAzureManagedIdentityToken,
+  getAzureWorkloadIdentityToken,
 } from './utils';
+import { env } from 'hono/adapter';
+import fs from 'fs';
```

Comment:
Note that I'm not an expert of Hono.
This is most likely very specific to the node runtime, not sure if we need to specify it in any way here.
Definitely open to feedback.